SimG4Core/Generators/interface/Generator.h

```diff
@@ -33,8 +33,7 @@ class Generator {
 
 private:
   bool particlePassesPrimaryCuts(const G4ThreeVector &p) const;
-  bool isExotic(int pdgcode) const;
-  bool isExoticNonDetectable(int pdgcode) const;
+  bool isChargedExotic(int pdgcode) const;
```

Comment:
@weishi10141993 , rename this to isExotic(int pdgcode)

Reply:
I didn't quite understand why/how the tracking of neutral extotic particle works at Geant4. What kind of interactions will be actually simulated since they are not charged and have no QCD interaction?

Reply:
@weishi10141993 , it may decay into charged secondaries.